net/dns/resolver/tsdns.go

```diff
@@ -83,6 +83,14 @@ type Config struct {
 	// LocalDomains is a list of DNS name suffixes that should not be
 	// routed to upstream resolvers.
 	LocalDomains []dnsname.FQDN
+	// ExitNodeBackupResolvers are where the local node when
+	// acting as an exit node and serving a DNS proxy should
+	// forward DNS requests to in the case where there are no
+	// routes found. For example, for Linux systemd-resolved
```

Comment:
"no routes" ? I don't understand.

Reply:
If the Tailnet DNS config has neither global resolvers or Split DNS routes.